sdk/core/azure_core_amqp/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 0.3.0 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 0.3.0 (2025-05-02)
 
 ### Other Changes
 
+- Updated dependencies.
```

Comment:
```suggestion
- Updated dependencies.
- Converted AMQP traits to use `async_trait` rather than attempting to implement the `async_trait` functionality manually
- Restructured and refactored AMQP errors to make them easier to interpret.
```